openpype/settings/entities/schemas/projects_schema/schemas/schema_houdini_general.json

```diff
@@ -0,0 +1,28 @@
+{
+    "type": "dict",
+    "key": "general",
+    "label": "General",
+    "collapsible": true,
+    "is_group": true,
+    "children": [
+        {
+            "type": "dict",
+            "collapsible": true,
+            "checkbox_key": "enabled",
+            "key": "job_path",
+            "label": "JOB Path",
+            "children": [
+                {
```

Comment:
Admittedly it's pretty unclear what the checkbox means really for `JOB Path` for an admin. Can imagine someone not fully grasping what enabling/disabling it would change. Should we add a more descriptive label inside the collapsible box?